gen/pbs_phys_cluster

```diff
@@ -39,16 +39,18 @@ my @facilities = $facilitiesAgent->getAssignedFacilities(service => $service->ge
 my @outputLines;
 
 my $sortingFunction = getSortingFunction("getName", 1);
+my %facilityNameAttribute;
 foreach my $facility (sort $sortingFunction @facilities) {
 
-	my $data = $servicesAgent->getHierarchicalData(service => $service->getId, facility => $facility->getId);
-	my %facilityAttributes = attributesToHash $data->getAttributes;
+	my $data = $servicesAgent->getHashedHierarchicalData(service => $service->getId, facility => $facility->getId);
+	my $facilityNameAttribute = $data->getFacilityAttributeValue(attrName => $A_FACILITY_NAME);
+	my $facilityPbsServerAttribute = $data->getFacilityAttributeValue(attrName => $A_FACILITY_PBS_SERVER);
 
-	if($facilityAttributes{$A_FACILITY_PBS_SERVER} ne $pbsServer) { next; }
+	if($facilityNameAttribute{$facilityPbsServerAttribute} ne $pbsServer) { next; }
```

Comment:
`$facilityNameAttribute` is not a structure. It represents just a string. You probably wanted to use `$facilityPbsServerAttribute ne $pbsServer` in the if clause

Reply:
Edited.